README.md

```diff
@@ -0,0 +1,344 @@
+# Building a Crypto News Website using the Microsoft Azure App Service and MongoDB Atlas
```

Comment:
Do we want to specify the language here as well so people coming in from Google know exactly what they are being dropped into?